yarn.lock

```diff
@@ -381,10 +349,10 @@
   dependencies:
     buffer "^5.4.3"
 
-"@bitgo/statics@^6.3.0-rc.1":
-  version "6.3.0-rc.1"
-  resolved "https://registry.yarnpkg.com/@bitgo/statics/-/statics-6.3.0-rc.1.tgz#b62b585c012836528d0fdf8156f0f51fe51ef50c"
-  integrity sha512-LkjkMSoUMXRjYQ0d5gqThNhLY7m2+8FRcE/lYJZ9QObT8kPcZpMNTkYVk2gE0eub4DldxI1UMvJ+8Y3Kny8bDg==
+"@bitgo/statics@^6.2.1":
+  version "6.2.1"
```

Comment:
Why is this getting downgraded?

Reply:
It was incorrectly upgraded to an release candidate in a merged PR. We should only ever have stable versions of packages inside this monorepo in the package.jsons and yarn.lock

Reply:
I don't see that noted in e.g. a commit message?

Reply:
It was changed during the merge conflict resolution here: https://github.com/BitGo/BitGoJS/pull/1071/commits/fffb9fbb030c815603b556b20cbc2cd6e2ff27c4